keras_nlp/models/roberta/roberta_tokenizer.py

```diff
@@ -68,10 +68,11 @@ class RobertaTokenizer(BytePairTokenizer):
     ...     merges=merges,
     ... )
     >>> tokenizer(inputs)
-    <tf.RaggedTensor [[5, 6, 7], [3, 4, 7]]>
+    <tf.RaggedTensor [[6, 7, 8], [4, 5, 8]]>
 
     Unbatched input.
-    >>> vocab = {"<s>": 0, "<pad>": 1, "</s>": 2, "Ġafter": 3, "noon": 4, "Ġsun": 5}
+    >>> vocab = {"<s>": 0, "<pad>": 1, "</s>": 2,  "<mask>": 3, "Ġafter": 4}
```

Comment:
`vocab.update({"noon": 5, "Ġsun": 6})`?

Reply:
Played around with this. I ended up liking most `vocab = {**vocab, "noon": 5, "Ġsun": 6}`. It's simpler than the line you commented on originally, but keep the vertical alignment, which makes this more easily scannable IMO.